### PR TITLE
fix (CloudFrontToS3 - use cloudfront function instead of lambda@edge)

### PR DESCRIPTION
**Pull request by hnishar**
*Issue #, if available:* [233](https://github.com/awslabs/aws-solutions-constructs/issues/233)

*Description of changes:*

Use CloudFront function instead of lambda@edge for CloudFrontToS3 construct

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by aws-solutions-constructs-team** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: codebuildgithubautobuildPro-fkVQbXRiQi6A
* Commit ID: 96e0f474b352902feecf319c7e69d194bacc5edb
* Result: FAILED
* [Build Logs](https://nis3u8ftb6.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=0cc3f81a-6657-4f3b-b9b4-48adc608f62c%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: source/patterns/@aws-solutions-constructs/aws-cloudfront-s3/lib/index.ts [hunk 1]
```diff
@@ -50,7 +49,7 @@ export interface CloudFrontToS3Props {
 
 export class CloudFrontToS3 extends Construct {
     public readonly cloudFrontWebDistribution: cloudfront.Distribution;
-    public readonly edgeLambdaFunctionVersion?: lambda.Version;
```
[the remark this answers is not in the document]
Comment (biffgaut) [reply source/patterns/@aws-solutions-constructs/aws-cloudfront-s3/lib/index.ts:53]: This is now a breaking change, isn't it? Although it seems extremely unlikely that a customer has used this property.

**Review by biffgaut — APPROVED** [review]

**Comment by aws-solutions-constructs-team** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: codebuildgithubautobuildPro-fkVQbXRiQi6A
* Commit ID: b51013b4713dc42e41e0b2e2c36e1d6de7970701
* Result: SUCCEEDED
* [Build Logs](https://nis3u8ftb6.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=18ab5528-37ab-4d8a-b6c2-01a2539e57c9%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by aws-solutions-constructs-team** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: codebuildgithubautobuildPro-fkVQbXRiQi6A
* Commit ID: 2595cd537a524d22c95360aeaaef58e40a89915e
* Result: SUCCEEDED
* [Build Logs](https://nis3u8ftb6.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=3e30c3ee-3c6f-4bae-9ea6-c73179c7109d%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Pull request merged**